Library/Homebrew/cmd/info.rb

```diff
@@ -412,4 +416,8 @@ def format_count(count)
   def format_percent(percent)
     format("%<percent>.2f", percent: percent)
   end
+
+  def linuxbrew?
```

Comment:
I deliberated for ages over this method name. We're moving away from the Linuxbrew name, but equally it describes the thing really well in this case I think.

Reply:
I think instead of a `linuxbrew?` method here this should use the similar `Library/Homebrew/extend/os/` override pattern. There could be `formula_path` and `analytics_path` methods or methods to get the formula or analytics json for a given formula.